lldb/test/CMakeLists.txt

```diff
@@ -174,6 +174,12 @@ llvm_canonicalize_cmake_booleans(
   LLVM_ENABLE_SHARED_LIBS
   LLDB_IS_64_BITS)
 
+# BEGIN SWIFT
+llvm_canonicalize_cmake_booleans(
```

Comment:
why isn't this conditional on having swift support enabled?
Ah.. this is the code that _declares_ LLDB_ENABLE_SWIFT_SUPPORT? My CMake is not very good.

Reply:
The GitHub diff doesn't show the full code above, but this code (and the one above) is 'canonicalizing' the bool variable so that we can use it in the lit.site.cfg.py.in file (it essentially just turns all the different strings CMake interprets as True or False into 1 and 0 respectively so that we can write the value into the Python file). Without that our config file would be turn the line `config.lldb_enable_swift = @LLDB_ENABLE_SWIFT_SUPPORT@` to `config.lldb_enable_swift = Off` (as `Off` is a legit 'false' value in CMake) and then Python would ask 'what does Off mean'? With this line the config file instead gets `config.lldb_enable_swift = 0` which Python understands.